modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/ArticleBodyRenderer.es.js

```diff
@@ -40,7 +40,7 @@ export default function ArticleBodyRenderer({
 		<>
 			{encodingFormat !== 'bbcode' && compactMode && (
 				<div
-					className={`questions-article-body-${id}`}
+					className={`questions-article-body questions-labels-limit ${id}`}
```

Comment:
Hmmm, I'm afraid of:

`questions-article-body-${id}`

becoming

`questions-article-body questions-labels-limit ${id}`

Because the first has -body-${id}

and the other limit ${id}

Which means, the `id` was appending with the text, but now is not.

this ID is necessary ?

Reply:
I checked in the other files, scss and etc.. if it is used. But, there is no connection.
I can keep the previous way like this: ```className={`questions-article-body-${id} questions-labels-limit`}```

Reply:
If it no causes any problem, it's ok :)